Classes/Private/Manager/Audio/OCTAudioEngine.m

```diff
@@ -385,6 +411,58 @@ - (BOOL)startGraph:(NSError **)error
     return YES;
 }
 
+- (BOOL)setupMaximumFramesPerSlice:(NSError **)error
+{
+
+    uint32_t maxSampleCount = kSampleCount + 1;
+    OSStatus status = _AudioUnitSetProperty(self.ioUnit,
+                                            kAudioUnitProperty_MaximumFramesPerSlice,
```

Comment:
I will remove this actually. This was there originally when I didn't have the microphone buffer implemented and wanted to get the inNumberOfFrames to give me an exact number. As to why 1920, this is one of the valid sample counts. There is no valid reason. 960, 1920, and 2880 are valid sample counts, and I picked whichever is closer.